src/chi-vue/src/views/DataTable/ClientSide/fixtures.ts

```diff
@@ -33,6 +33,13 @@ export const exampleConfig = {
   truncation: true,
 };
 
+export const bulkActionConfig = {
```

Comment:
Is all of this necessary?

Reply:
Not all of them actually. I set few of them as placeholders, will remove not needed ones.

Reply:
Please see the previous comment on internal communication between DataTable and BulkActions. This should not be needed.